cmd/create/cluster/runner.go

```diff
@@ -98,7 +98,6 @@ func (r *runner) run(ctx context.Context, _ *cobra.Command, _ []string) error {
 			// Create a keypair and kubeconfig for the new tenant cluster
 			err := gsClient.CreateKubeconfig(ctx, clusterID, kubeconfigPath)
 			if err != nil {
-				// TODO: check to see if it's a permanent error or the kubeconfig just isn't ready yet
```

Comment:
If/when this occurs, let's capture the error to somehow match it here. But this doesn't need to stay IMO if this isn't ongoing.